apps/admin/kube-slack/kube-slack.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
 spec:
   releaseName: kube-slack
   values:
+    valuesFrom:
```

Comment:
Isn’t valuesFrom part of the spec? Values is passed through to the chart not flux to retrieve from the sops secret

Reply:
kured chart is using spec.values.valuesFrom and is deploying okay? https://github.com/hmcts/sds-flux-config/blob/master/apps/admin/kured/kured.yaml